index.js

```diff
@@ -528,6 +535,29 @@ function fastifyMultipart (fastify, options, done) {
     return requestFiles
   }
 
+  async function * filesFromFields (container) {
```

Comment:
Why async generator but not a generator?
It doesn't really need to be async.

Reply:
Already changed